includes/settings-functions.php

```diff
@@ -3,161 +3,8 @@
 	exit; // Exit if accessed directly.
 }
 
-add_action( 'admin_menu', 'hello_elementor_settings_page' );
 add_action( 'init', 'hello_elementor_tweak_settings', 0 );
```

Comment:
@nicoladj77 maybe move all the logic into a new `Settings` component and leave just wrapper functions that will cll the component's methods?